torch/onnx/symbolic_opset9.py

```diff
@@ -1,36 +1,30 @@
+"""This file exports ONNX ops for opset 9.
+
+Opset 9 is supported by ONNX release 1.4.1
+release on 01/23/19
+"""
+
+import functools
 import math
+import sys
 import warnings
-from functools import partial, wraps
-from sys import maxsize as maxsize
 from typing import Optional
 
 import torch
+import torch.nn.modules.utils as modules_utils
 import torch.onnx
-import torch.onnx.symbolic_helper as sym_help
-from torch._C import DeviceObjType, ListType, OptionalType
-from torch.nn.modules.utils import _pair, _single, _triple
+from torch import _C
 
 # This import monkey-patches graph manipulation methods on Graph, used for the
 # ONNX symbolics
 from torch.onnx import _patch_torch  # noqa: F401
+from torch.onnx import symbolic_helper
 from torch.onnx._globals import GLOBALS
-from torch.onnx.symbolic_helper import (
-    ScalarType,
-    _parse_arg,
-    _unimplemented,
-    args_have_same_dtype,
-    parse_args,
-    quantized_args,
-)
+from torch.onnx.symbolic_helper import _parse_arg, parse_args, quantized_args
```

Comment:
how do you decide when to import only modules and when to relax the constraint?

Reply:
Good question. My initial thought was that they are decorators so maybe people are used to type less. Thinking again I should probably not make the exception and remove this line